.gitignore

```diff
@@ -40,6 +40,7 @@ tmp/
 npm-debug.log*
 yarn-error.log*
 .ng_pkg_build/
+.ng-dev.log
```

Comment:
You should additionally add `.ng-dev.user*` so that users can have local configs if desired.